scripts/configure/zowe-configure-api-mediation.sh

```diff
@@ -21,6 +21,12 @@
 # $ZOWE_APIM_EXTERNAL_CERTIFICATE_AUTHORITIES
 # $ZOWE_APIM_VERIFY_CERTIFICATES
 
+###
```

Comment:
My dodgy suggestion to try and make the process backward compatible in the short term is that this script should be re-purposed to do the following logic:
- Create a new yaml field for the keystore directory - default to /global/zowe/keystore or whatever.
- Using the yaml file variables that it already has access to - create a new temporary `zowe-setup-certificates.env` file and inject them in. 
- Call `bin/zowe-setup-certificates.sh` passing in this temp file
- Clean up the temp file
- Update the instance.env sed to point to the new keystore directory

We might need to look at the `zowe-configure-zlux-certificates.sh` and see what variables it uses and whether they are exported, but it's a short term like for like replacement I think. After the zlux componentisation is complete this file can be deleted and we'll default to the 'proper' behaviour